packages/node_modules/@webex/webex-core/src/lib/services/services.js

```diff
@@ -394,14 +403,14 @@ const Services = WebexPlugin.extend({
             'x-prelogin-userid': undefined
           },
           body: {
-            email, reqId, countryCode, timeZone: timezone
+            email, reqId, countryCode, timeZone: timezone, ...activationOptions
```

Comment:
new params for readability
```suggestion
            email,
            reqId,
            countryCode,
            timeZone: timezone,
            ...activationOptions
```